packages/testing/src/api/index.ts

```diff
@@ -32,36 +35,63 @@ class FakeCredentials {
   }
 }
 
-export function apps(): (admin.app.App | null)[] {
+export function adminApps(): (admin.app.App | null)[] {
   return admin.apps;
 }
 
-export function initializeAdminApp(options: any): admin.app.App {
-  if (!('databaseName' in options)) {
-    throw new Error('databaseName not specified');
-  }
+export function apps(): (FirebaseApp | null)[] {
+  return firebase.apps;
+}
+
+type AdminAppOptions = {
+  databaseName: string;
+}
+
+export function initializeAdminApp(options: AdminAppOptions): admin.app.App {
+  const appName = 'app-' + new Date().getTime() + '-' + Math.random();
   return admin.initializeApp(
     {
       credential: new FakeCredentials(),
       databaseURL: DBURL + '?ns=' + options.databaseName
     },
-    'app-' + (new Date().getTime() + Math.random())
+    appName
   );
 }
 
-export function initializeTestApp(options: any): admin.app.App {
-  if (!('databaseName' in options)) {
-    throw new Error('databaseName not specified');
+export type DatabaseAppOptions = {
+  databaseName: string;
+  auth: object;
+}
+export type FirestoreAppOptions = {
+  projectId: string;
+  auth: object;
+}
+export function initializeTestApp(options: DatabaseAppOptions | FirestoreAppOptions): FirebaseApp {
+  let appOptions: FirebaseOptions;
+  if ('databaseName' in options) {
+    appOptions = {
+      databaseURL: DBURL + '?ns=' + options.databaseName
+    };
+  } else if ('projectId' in options) {
```

Comment:
Is there a reason to not support both `projectId` and `databseURL`? This code might be a little cleaner if you supported both, even if the interface doesn't allow for it.